src/AutocompleteBadges.soy

```diff
@@ -0,0 +1,21 @@
+{namespace AutocompleteBadges}
+
+/**
+ * This renders the component's whole content.
+ * @param? badges
+ */
+{template .render}
+
+	<div class="autocomplete-badges">
+		<textarea autocomplete="off" class="autocomplete-field form-control" wrap="off" style="height: 30px; resize: none;"></textarea>		
```

Comment:
We like to keep metal components as free from custom css as possible, so that it works but doesn't try to style things that users will want to remove later. Having css in a style attribute makes it even harder for developers to change it if they wish, so in this case it's best to just remove the attribute completely, and let developers style it as they wish.